src/main/java/redis/clients/jedis/resps/ConsumerFullInfo.java

```diff
@@ -0,0 +1,53 @@
+package redis.clients.jedis.resps;
+
+import java.io.Serializable;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * This class holds information about a stream consumer with command <code>xinfo stream mystream full<code/>.
+ * They can be access via getters. For future purpose there is also {@link #getConsumerInfo()} method that
+ * returns a generic {@code Map} - in case where more info is returned from the server.
+ */
+public class ConsumerFullInfo implements Serializable {
```

Comment:
Sorry for saying this late. But `StreamConsumerFullInfo` may be a better name here.

Reply:
done